test/js-api/limits.any.js

```diff
@@ -250,7 +250,16 @@ function testModuleSizeLimit(size, expectPass) {
 
   // Define a WebAssembly module that consists of a single custom section which
   // has an empty name. The module size will be `size`.
-  const buffer = new Uint8Array(size);
+  let buffer;
+  try {
+    buffer = new Uint8Array(size);
+  } catch (e) {
+    if (e instanceof RangeError) {
+      // 32-bit systems may fail to allocate a big TypedArray.
```

Comment:
I would just say `TypedArray allocation may fail`. The current comment seems V8-specific.

Reply:
Ack, I removed the mention of 32-bit systems.